desktop/components/main_layout/header/view.coffee

```diff
@@ -38,6 +39,8 @@ module.exports = class HeaderView extends Backbone.View
       mode: 'suggest'
       limit: 7
 
+    splitTest('highlight_header_search').view()
```

Comment:
I'm wondering if this is where I should be calling the view method. I figured putting it after the search bar was instantiated made sense, but wasn't sure. Thoughts?